charts/osschallenge/values.yaml

```
@@ -0,0 +1,108 @@
+    internalPort: 8000
+  # osschallenge.resources -- Resource limits for osschallenge
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+    #  memory: 128Mi
+  livenessProbe:
+    # osschallenge.livenessProbe.enabled -- Enable liveness probe on osschallenge
+    enabled: true
+    # osschallenge.livenessProbe.initialDelaySeconds -- Number of seconds after the container has started before liveness probe is initiated
+    initialDelaySeconds: 60
+    # osschallenge.livenessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # osschallenge.livenessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # osschallenge.livenessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # osschallenge.livenessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  readinessProbe:
+    # osschallenge.readinessProbe.enabled -- Enable readiness probe on osschallenge
+    enabled: true
+    # osschallenge.readinessProbe.initialDelaySeconds -- Number of seconds after the container has started before readiness probe is initiated
+    initialDelaySeconds: 30
+    # osschallenge.readinessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # osschallenge.readinessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # osschallenge.readinessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # osschallenge.readinessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  settings:
+    # django settings for osschallenge
+    # osschallenge.settings.emailHost -- Connection string of SMTP server to send mails
+    emailHost: smtp.example.com
+    # osschallenge.settings.emailPort -- Port for the SMTP server
+    emailPort: 587
+    # osschallenge.settings.emailFrom -- Default email address to use for various responses
+    emailFrom: osschallenge@oss-challenge.ch
+    # osschallenge.settings.emailUser -- Username for the SMTP server
+    emailUser: example@example.com
+    # osschallenge.settings.emailPassword -- Password for the SMTP server
+    emailPassword: ~
+    # osschallenge.settings.siteUrl -- URL for OSS-Challenge
+    siteUrl: oss-challenge.ch
```

This should either be a real URL or be called `stiteHostname` or similar.
```suggestion
    siteUrl: https://oss-challenge.ch
```
```suggestion
    siteHostname: oss-challenge.ch
```

---

Did you re-test oss-challenge after changing this?

---

I know that the URL is currently in use by on old oss-challenge vm, if that's what you meant.